packages/workflow/src/Extensions/StringExtensions.ts

```diff
@@ -407,7 +407,8 @@ toSnakeCase.doc = {
 
 toTitleCase.doc = {
 	name: 'toTitleCase',
-	description: 'Formats a string to title case. Example: "This Is a Title".',
+	description:
+		'Formats a string to title case. Example: "This Is a Title". Will not change already uppercaes letters to prevent losing information from acronyms and trademarks such as iPhone or FAANG.',
```

Comment:
```suggestion
		'Formats a string to title case. Example: "This Is a Title". Will not change already uppercase letters to prevent losing information from acronyms and trademarks such as iPhone or FAANG.',
```